This patch silences a spurious warning that occurs when running the GUI under VirtualGL
with EGL enabled.
From: github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/ParaView/ParaView-5.11.1-remove_glew_init_warning.patch
Update: Cintia Willemyns (Vrije Universiteit Brussel)
--- ParaView-v5.13.2.orig/VTK/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx	2024-12-30 00:41:05.000000000 +0100
+++ ParaView-v5.13.2/VTK/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx	2025-02-12 10:27:08.000000000 +0100
@@ -705,8 +705,8 @@
     this->GlewInitValid = (result == GLEW_OK);
     if (!this->GlewInitValid)
     {
-      const char* errorMsg = reinterpret_cast<const char*>(glewGetErrorString(result));
-      vtkErrorMacro("GLEW could not be initialized: " << errorMsg);
+      //const char* errorMsg = reinterpret_cast<const char*>(glewGetErrorString(result));
+      //vtkErrorMacro("GLEW could not be initialized: " << errorMsg);
       return;
     }
 
